models/aeif_cond_alpha_multisynapse.cpp

```diff
@@ -65,6 +67,67 @@ RecordablesMap< aeif_cond_alpha_multisynapse >::create()
 }
 
 /* ----------------------------------------------------------------
+ * Right-hand side function
+ * ---------------------------------------------------------------- */
+
+extern "C" int
+aeif_cond_alpha_multisynapse_dynamics( double,
+  const double y[],
+  double f[],
+  void* pnode )
+{
+  // y[] is the state vector supplied by the integrator,
+  // not the state vector in the node, node.S_.y[].
+
+  typedef nest::aeif_cond_alpha_multisynapse::State_ S;
+
+  // get access to node so we can almost work as in a member function
+  assert( pnode );
+  const nest::aeif_cond_alpha_multisynapse& node =
+    *( reinterpret_cast< nest::aeif_cond_alpha_multisynapse* >( pnode ) );
+
+  const bool is_refractory = node.S_.r_ > 0;
```

Comment:
I fear we still have a misunderstanding: now, via your modification of ``S_.r_`` in ``update``, the ``t_ref = 0`` test passes, but the discrepancy between this model and the refractory LSODAR implementation still exists at spiketime because if ``t_ref > 0``, then we get the refractory condition during the "spiketime step", which we do not want since it modifies the spike trajectory.